Source/WebCore/rendering/RenderReplaced.h

```diff
@@ -74,10 +74,11 @@ class RenderReplaced : public RenderBox {
 
     void willBeDestroyed() override;
 
+    virtual RenderBox* embeddedContentBox() const { return 0; }
+
 private:
     LayoutUnit computeConstrainedLogicalWidth(ShouldComputePreferred) const;
 
-    virtual RenderBox* embeddedContentBox() const { return 0; }
```

Comment:
Why did you make this method public? I don't see anything in the patch that requires it to become public.

Reply:
It is protected and not private. Without this, it leads to compiler error.

Reply:
Unclear to me this needs to be protected, I don't see any new calls to `embeddedContentBox()` here.

Reply:
Oh I see, RenderWidget calls embeddedContentBox but you're moving it away from RenderWidget to its subclass.